zstacklib/zstacklib/utils/upload_task.py

```
@@ -0,0 +1,409 @@
+    @lock.lock(lock_name)
+    def get_task(self, image_uuid):
+        # type: (str) -> UploadTask
+        return self.tasks.get(image_uuid)
+
+    def upload(self, req):
+        upload_param = self.get_upload_param(req.headers)
+        task = self.get_upload_task(upload_param)
+        self.upload_slice(req.body, upload_param, task)
+
+        if task.allow_image_completing():
+            pool, img_name = task.dstPath.split('/')
```

Ruff (0.13.1)

Unpacked variable `img_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)